source/_patterns/00-atoms/references/reference.mustache

```diff
@@ -39,11 +41,23 @@
     {{> atoms-doi}}
   {{/doi}}
 
+  {{^doi}}
+
+    {{#titleLink}}
+      <div class="reference__link-outer">
+        <a href="{{titleLink}}" class="reference__link">{{{titleLink}}}</a>
+      </div>
+    {{/titleLink}}
+
+  {{/doi}}
+
   {{#hasAbstracts}}
     <ul class="reference__abstracts">
   {{/hasAbstracts}}
   {{#abstracts}}
-    <li class="reference__abstract"><a href="{{url}}" class="reference__abstract_link">{{{name}}}</a></li>
+    <li class="reference__abstract">
```

Comment:
Undo this change

Reply:
Can you explain why?

Reply:
Can you explain why as I have just moved the link to the line below as just tidying up the code.